### ISSUE TITLE
Infer TTS model.

### ISSUE THREAD

**Issue by swoboo** [ictnlp/DASpeech#6]
To infer the fastspeech2, I try:
fairseq-generate data/cvss-c/fr-en/tts/ \
    --config-yaml config.yaml  --gen-subset test \
    --task text_to_speech --criterion fastspeech2 --arch fastspeech2 \
    --path $checkpoint_dir/average_best_checkpoint.pt \
    --encoder-layers 4 --encoder-embed-dim 256 --encoder-attention-heads 4 \
    --decoder-layers 4 --decoder-embed-dim 256 --decoder-attention-heads 4 \
    --fft-hidden-dim 1024 \
    --required-batch-size-multiple 1 ... 
but it's not work.how do I perform inference with a pre-trained fastspeech2 model?

**Comment by Poeroz** [comment]
Hi, please try the following script.

```
python DASpeech/generator/generate_features.py \
  data/cvss-c/fr-en/tts/ \
  --user-dir DASpeech \
  --config-yaml config.yaml --gen-subset test --task text_to_speech \
  --path $ckpt_pth --max-tokens 40000 --spec-bwd-max-iter 32 \
  --required-batch-size-multiple 1 \
  --results-path $results_path \
  --generator-type nat_tts
```

**Comment by swoboo** [comment]
Thanks~！
